### PR TITLE
Model book

### PR DESCRIPTION
**Pull request by rodrigofranc**
### Summary

- Book Model
- Getters and Setters
- Contructor

### Trello Card

[https://trello.com/c/YiSaA8Ne/19-crear-modelo-de-book](https://trello.com/c/YiSaA8Ne/19-crear-modelo-de-book)

### CODE REVIEW

File: src/main/java/wolox/training/TrainingApplication.java [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+
+@EntityScan("org.wolox.training.models.Book")
```
Comment (zeminlu) [inline src/main/java/wolox/training/TrainingApplication.java]: This is only necessary if the entity is outside of the java package, meaning, src/main/java.
Given yours is in it, it should work without this. Check it out!

File: src/main/java/wolox/training/models/Book.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import javax.persistence.Column;
+
+import javax.persistence.Id;
+
+import javax.persistence.Entity;
+@Entity
+public class Book {
+
+
+    @Column(nullable = true)
```
Comment (zeminlu) [inline src/main/java/wolox/training/models/Book.java]: Not necessary, it's nullable by default.

File: src/main/java/wolox/training/models/Book.java [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    private String Genre;
+
+    @Column(nullable = false)
+    private String Author;
```
Comment (zeminlu) [inline src/main/java/wolox/training/models/Book.java:16]: All these variables should start lowercase.

**Review by zeminlu — CHANGES_REQUESTED** [review]
Looks good! Just a couple of changes and you're good to go!

**Review by rodrigofranc — COMMENTED** [review]
Delete unnecessary annotations on book model

**Review by zeminlu — APPROVED** [review]

**Pull request merged**